domains/vinicius.json

```diff
@@ -6,6 +6,6 @@
     "email": "59481808+vncsmnl@users.noreply.github.com"
```

Comment:
Sorry, you can't use this email as we cannot contact you through a noreply email.

Reply:
fixed